### PR TITLE
Run all EDA notebooks to update age in years 

### PR DESCRIPTION
**Pull request by jasminziegler**
# Description

Issue #336 round(age) has been exchanged with flore(age)
Now we want to run all EDA notebooks to have the updated age in years figures.

User story #336

# How Has This Been Tested?
ran notebook and checked age calculations for some datapoints

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] My Code is review by 1 People

### CODE REVIEW

**Comment by jasminziegler** [comment]
issue with datasets, in clarification with team

File: src/data_utils/dataset_EDA/anon-depthmap-testset/EDA_anon-depthmap-testset.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (jasminziegler) [inline src/data_utils/dataset_EDA/anon-depthmap-testset/EDA_anon-depthmap-testset.ipynb:2]: <p>that's funny that now it is not colorful anymore. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Welthungerhilfe/cgm-ml/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Comment by jasminziegler** [comment]
Shanky migrated datasets today, are all available now with new connection file format that was created to prevent datasets from vanishing. I wrote down all the steps I had to do to gather all information needed for the new connection file and sent that to Shanky. He will soon add this to the wiki.

**Comment by jasminziegler** [comment]
ran all notebooks successfully except for rgbd5k, labels folder and scans.csv file missing. in clarification with team

**Review by markus-hinsche — CHANGES_REQUESTED** [review]
Please undo the changes to `src/data_utils/dataset_EDA/anon-rgbd-5k/EDA_anon-rgbd-5kscans.ipynb` since we would keep the rough information about this dataset. 

After this, we can merge

**Pull request merged**